src/models/v2/message.ts

```diff
@@ -68,7 +68,7 @@ export class Message extends MessageTrait<v2.MessageObject> implements MessageIn
           (operation.message.oneOf || []).includes(this._json)
         )) {
           operations.push(
-            this.createModel(Operation, operation, { pointer: `/channels/${tilde(channelAddress)}/${operationAction}`, action: operationAction })
+            this.createModel(Operation, operation, { id: '', pointer: `/channels/${tilde(channelAddress)}/${operationAction}`, action: operationAction as 'subscribe' | 'publish' })
```

Comment:
Can we add a new type for the Operation actions for v2 ('subscribe' | 'publish') so we can avoid "hardcoding" this here but instead use such type?

Reply:
Good point, I will add! :)

Reply:
Done.